Jenkinsfile

```diff
@@ -69,6 +69,11 @@ pipeline {
                     echo 'testing coscrad-frontend'
                     sh 'npx nx test coscrad-frontend --skip-nx-cache'
 
+                    // See https://github.com/facebook/jest/issues/11354
+                    sh "sed -i -e 's/const FORCE_EXIT_DELAY = 500;/const FORCE_EXIT_DELAY = 7000;/g' ./node_modules/jest-worker/build/base/BaseWorkerPool.js"
```

Comment:
Whoa, this is a bit wiggy, lol

Reply:
Haha. Yea see the thread in Discord for more info about why it's necessary to patch Jest this way. If they expose this const as a config option, I'll get rid of this magic. One thing I think about is potentially splitting up our test suite into smaller batches and running these batches in parallel. But obviously this one liner is a little bit less of my time.